pkg/ingester/ingester_v2.go

```diff
@@ -1534,6 +1557,11 @@ func (i *Ingester) getOrCreateTSDB(userID string, force bool) (*userTSDB, error)
 		}
 	}
 
+	// If all the TSDB's are in the process of closing, then should not proceed to creating a new TSDB
```

Comment:
Please remove this change. Same comment from above applies.

Reply:
Good catch, I've removed it.